metrics/cgroups/v1/memory.go

```diff
@@ -543,7 +543,7 @@ var memoryMetrics = []*metric{
 		unit: metrics.Total,
 		vt:   prometheus.GaugeValue,
 		getValues: func(stats *v1.Metrics) []value {
-			if stats.Memory == nil {
+			if stats.GetMemory().GetUsage() == nil {
 				return nil
 			}
```

Comment:
I guess, this will panic if `stats.GetMemory` returns nil? should we check `stats.GetMemory()` nility before?

Reply:
The accessors grpc generates makes this safe afair, as they're pointer receivers that check the nility inside. They generate essentially the same as -> https://go.dev/play/p/DIaC-ZyGB1l

Reply:
Ohhhh TIL! ty @dcantah